diff --git a/arch/arm/mach-meson8b/cpu.c b/arch/arm/mach-meson8b/cpu.c
index 91dffe5..1641494 100755
--- a/arch/arm/mach-meson8b/cpu.c
+++ b/arch/arm/mach-meson8b/cpu.c
@@ -84,3 +84,11 @@ int get_meson_cpu_version(int level)
 	return 0;
 }
 EXPORT_SYMBOL(get_meson_cpu_version);
+
+int (*get_cpu_temperature_celius)(void) = NULL;
+EXPORT_SYMBOL_GPL(get_cpu_temperature_celius);
+
+int get_cpu_temperature(void)
+{
+	return get_cpu_temperature_celius ? get_cpu_temperature_celius() : -1;
+}
