diff --git a/drivers/thermal/thermal_core.c b/drivers/thermal/thermal_core.c
index 399155f..40d0df9 100755
--- a/drivers/thermal/thermal_core.c
+++ b/drivers/thermal/thermal_core.c
@@ -36,7 +36,7 @@
 #include <net/netlink.h>
 #include <net/genetlink.h>
 
-#include <linux/thermal_core.h>
+#include "thermal_core.h"
 
 MODULE_AUTHOR("Zhang Rui");
 MODULE_DESCRIPTION("Generic thermal management sysfs support");
diff --git a/drivers/thermal/user_space.c b/drivers/thermal/user_space.c
index a5cbf7d..10adcdd 100644
--- a/drivers/thermal/user_space.c
+++ b/drivers/thermal/user_space.c
@@ -24,7 +24,7 @@
 
 #include <linux/thermal.h>
 
-#include <linux/thermal_core.h>
+#include "thermal_core.h"
 
 /**
  * notify_user_space - Notifies user space about thermal events
